### ISSUE TITLE
NuxtJs Configuration

### ISSUE THREAD

**Issue by bekee** [beyondcode/laravel-websockets#567]
I've had a hard time configuring this package to work with nuxtjs on my local system, please I need assistance.

**Comment by abo3ish** [comment]
I am using Laravel-WebSockets with Nuxtjs and it's working fine on the localhost although I have some problems in the deployment. 

I am using https://github.com/nuxt-community/laravel-echo 

and here is my localhost configuration in nuxt.config.js
`buildModules: [
    '@nuxtjs/laravel-echo'
  ],

  echo: {
    broadcaster: 'pusher',
    plugins: [
      '~/plugins/echo.js' // create a plugin file and you can find the code in the link above
    ],
    key: 'pusher_key',
    wsHost: 'localhost', // or your host domain
    wsPort: 6001,
    connectOnLogin: true,
    disconnectOnLogout: true
  },`

**Comment by bekee** [comment]
@abo3ish  please what version of laravel-echo, pusher and laravel-websocket are you using?

**Comment by abo3ish** [comment]
package.json
```
"@nuxtjs/laravel-echo": "^1.1.0",
"pusher-js": "^4.3.1"
```

composer.json

```
"beyondcode/laravel-websockets": "^1.4",
"pusher/pusher-php-server": "^4.1"
```

**Comment by VinceSanityyy** [comment]
this is not working for me @bekee . Even if i fire an event in websocket admin, nuxt wont recieve the event

**Comment by bekee** [comment]
Sorry for the late response. I used same configuration see below;

`echo: {
    broadcaster: 'pusher',
    plugins: [
      '~/plugins/mixins/echo.js' // create a plugin file and you can find the code in the link above
    ],
    key: 'anyKeyOfChoice',
    wsHost: 'socket.xxxxx.com', // or your host domain --- created a proxy to the socket with nginx
    wsPort: 443, //enable ssl via letsencrypt
    connectOnLogin: true,
    disconnectOnLogout: true,
    authEndpoint: 'http://api.travel.test/broadcasting/auth', // you can connect auth service of your application
  },`


In short I created and hosted this module in a separate laravel application then I can connect to it via the wsHost key from any of my apps.

Hope this helps
